gtsfm/two_view_estimator.py

```diff
@@ -25,6 +30,13 @@
 pil_logger = logging.getLogger("PIL")
 pil_logger.setLevel(logging.INFO)
 
+BA_NOISE_MODEL_EPIPOLAR_ALGEBRAIC_ERROR = gtsam.noiseModel.Isotropic.Sigma(1, 0.01)
```

Comment:
Thanks for the PR, Ayush. Very cool to see the improvement.

Do we use `BA_NOISE_MODEL_EPIPOLAR_ALGEBRAIC_ERROR` anywhere?

Reply:
removed it.